tests/acceptance/test_dcos_package.py

```diff
@@ -15,7 +15,16 @@ def test_uninstall_package_and_wait():
 
 def test_install_package_with_json_options():
     install_package_and_wait('chronos', None, 'big-chronos', None, {"chronos": {"cpus": 2}})
-    assert get_service_task('marathon', 'big-chronos')['resources']['cpus'] == 2
+    match = False
+    now = time.time()
+    future = now + 60
+    while now < future:
+        if 'resources' in get_service_task('marathon', 'big-chronos'):
+            if get_service_task('marathon', 'big-chronos')['resources']['cpus'] == 2:
+                match = True
+        time.sleep(1)
+        now = time.time()
+    assert match
```

Comment:
if we were to keep this (because it actually waits for a resource)... then we should do:

```
def task_resource_predicate():
  task = get_service_task('marathon', 'big-chronos')
  return task['resources']['cpus'] == 2

wait_for(task_resource_predicate)
```